### PR TITLE
Release v1.13.0

### PR DESCRIPTION
**Pull request by arminru**
This PR prepares the changelog for this month's spec release and adds some missing entries.

### CODE REVIEW

**Review by bogdandrutu — DISMISSED** [review]
Please resolve https://github.com/open-telemetry/opentelemetry-specification/issues/2726 before, to not release the new direction changes without a clear guidance.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,19 +11,56 @@ release.
...
+
+### OpenTelemetry Protocol
+
+### SDK Configuration
+
+### Telemetry Schemas
+
+### Common
+
+## v1.13.0 (2022-08-16)
```
Comment (arminru) [inline CHANGELOG.md:32]: // TODO update before merging

**Comment by tigrannajaryan** [comment]
> Please resolve #2726 before, to not release the new direction changes without a clear guidance.

@bogdandrutu do you want full resolution of #2726 or it is enough to if we merge https://github.com/open-telemetry/opentelemetry-specification/pull/2748 which reverts #2617 and we can work on resolving #2726 fully after the release (including adding better guidance about usage of attributes vs separate metrics)?

**Review by reyang — APPROVED** [review]

**Comment by reyang** [comment]
> Please resolve #2726 before, to not release the new direction changes without a clear guidance.

@bogdandrutu this should have been resolved by #2748, PTAL.

**Review by carlosalberto — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
Note that https://github.com/open-telemetry/opentelemetry-specification/issues/2762 is a release blocker.

File: CHANGELOG.md (hunk 1 above)
Comment (carlosalberto) [reply CHANGELOG.md:32]:
```suggestion
## v1.13.0 (2022-09-14)
```

**Comment by carlosalberto** [comment]
@open-telemetry/specs-approvers The last blocking item has been merged - I suggest we release later today. Please take a look, etc.

**Review by bogdandrutu — APPROVED** [review]

**Review by jmacd — APPROVED** [review]

**Comment by carlosalberto** [comment]
@tigrannajaryan Makes sense - early Friday as release date (as long as no complains happen) sounds good enough?

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,19 +11,56 @@ release.
...
+
+### OpenTelemetry Protocol
+
+### SDK Configuration
+
+### Telemetry Schemas
+
+### Common
+
+## v1.13.0 (2022-09-14)
```
Comment (carlosalberto) [inline CHANGELOG.md:32]:
```suggestion
## v1.13.0 (2022-09-16)
```

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**